src/internal/m365/collection/groups/groups_handler.go

```diff
@@ -0,0 +1,58 @@
+package groups
+
+import (
+	"context"
+
+	"github.com/alcionai/corso/src/pkg/selectors"
+	"github.com/alcionai/corso/src/pkg/services/m365/api"
+	"github.com/microsoft/kiota-abstractions-go/serialization"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+)
+
+var _ BackupHandler = &groupBackupHandler{}
+
+type groupBackupHandler struct {
+	ac      api.Channels
+	groupID string
+	scope   selectors.GroupsScope
```

Comment:
is `scope` used at all here?  If not, it can be removed.  We can always add it back in later if necessary.